src/components/MyKiva/NextYearGoalCard.vue

```diff
@@ -18,9 +18,9 @@
 					> {{ prevYearLoans }} women</span><br>shape their futures!
 				</h3>
 				<h3 class="tw-text-center" v-else>
-					Lenders helped <span
+					Lenders like you help <span
```

Comment:
Wondering if could be a computed property and use `v-html` for just one `h3` tag.